CHANGELOG.md

```diff
@@ -1,23 +1,61 @@
-# 2.0.0 (Breaking update)
+# Changelog
 
-## Additions
+- [Version 3.0.0](#version-300)
+- [Version 2.0.0](#version-200)
 
-* New `command` input
-    * This allows you to specify the Wrangler command you would like to run.
+## Version 3.0.0 (Breaking update)
+
+### Additions
+
+- **Rewritten Wrangler Action in TypeScript.**
+- Bulk secrets API utilization from Wrangler.
+- Added testing for improved reliability.
+- Implemented improved command functionality.
+- Now using Node for the Action engine/runner.
+- Open discussions with the community on all changes through GitHub Discussions and monitored Issues.
+
+### Removals
+
+- Removed Docker as a dependency.
+- Dropped support for Wrangler v1.
+
+### Changes
+
+- Fixed CI/CD issues.
+
+### Breaking changes
+
+- Docker is no longer a dependency.
```

Comment:
This isn't a breaking change

Reply:
I wasn't 100% certain but I will take your word on it lol